app/models/manageiq/providers/vmware/infra_manager/ems_folder.rb

```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::Vmware::InfraManager::EmsFolder < EmsFolder
```

Comment:
@agrare Are we going to need a data migration for this new class?  This will seriously break refresh if we don't, I think.

Reply:
I think so. Working on it.

Reply:
@Fryguy I _think_ the existing folders will be found and have their type updated on the first refresh after upgrade without being destroyed but let me confirm that.

Reply:
Ah never mind, forgot we `.except` the `:type` in `save_inventory_with_findkey`

Reply:
I have to delete the existing folders then the refresher would add them back in with type populated. Seems the refresher does not update the existing ones.

Reply:
@lfu yeah that's because of https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_refresh/save_inventory_helper.rb#L101